tardis/conftest.py

```diff
@@ -117,3 +119,29 @@ def included_he_atomic_data(test_data_path):
 def tardis_config_verysimple():
     return yaml_load_config_file(
         'tardis/io/tests/data/tardis_configv1_verysimple.yml')
+
+###
+# HDF Fixtures
+###
+
+@pytest.fixture(scope="session")
+def hdf_file_path(tmpdir_factory):
+    path = tmpdir_factory.mktemp('hdf_buffer').join('test.hdf')
+    return str(path)
+
+@pytest.fixture(scope="session")
+def hdf_config():
+    filename = 'tardis_configv1_verysimple.yml'
+    path = os.path.abspath(os.path.join('tardis/io/tests/data/', filename))
+    config = Configuration.from_yaml(path)
+    return config
+
+@pytest.fixture(scope="session")
+def model(hdf_config):
+    model = Radial1DModel.from_config(hdf_config)
+    return model
+
+@pytest.fixture(scope="session")
+def homologous_density(hdf_config):
```

Comment:
I would not say that homologous density is used anywhere else than the corresponding test